frontend/src/features/admin-form/responses/FeedbackPage/queries.ts

```diff
@@ -0,0 +1,27 @@
+import { useQuery, UseQueryResult } from 'react-query'
+import { useParams } from 'react-router-dom'
+
+import { FormFeedbackMetaDto } from '~shared/types/form'
+
+import { adminFormKeys } from '../../common/queries'
+
+import { getFormFeedback } from './FeedbackService'
+
+export const adminFormFeedbackKeys = {
+  base: [...adminFormKeys.base, 'feedback'] as const,
+  id: (id: string) => [...adminFormFeedbackKeys.base, id] as const,
+}
+
+/**
+ * @precondition Must be wrapped in a Router as `useParam` is used.
+ */
+export const useFormFeedback = (): UseQueryResult<FormFeedbackMetaDto> => {
+  const { formId } = useParams()
+  if (!formId) throw new Error('No formId provided')
+
+  return useQuery(
+    adminFormFeedbackKeys.id(formId),
+    () => getFormFeedback(formId),
+    { staleTime: 10 * 60 * 1000 },
```

Comment:
Probably no need to prematurely optimize with caching

Reply:
0e643c3